### DISCUSSION TITLE
Obtaining a parameter set for a model

### DISCUSSION THREAD

**Discussion by MatHunt** [pybamm-team/PyBaMM#2407] [Q&A] [answered]
Hi,

I'm new to using Pybamm, and I have been following the tutorials to get a feeling for how to use it, and I've been getting the syntax mostly. I want to change one or two of the parameters but the command to check for a certain value parameter_values = pybamm.ParameterValues("Chen2020") comes up with the following error:
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
Cell In [8], line 1
----> 1 parameter_values = pybamm.ParameterValues("Chen2020")

File ~\AppData\Local\Programs\Python\Python310\lib\site-packages\pybamm\parameters\parameter_values.py:76, in ParameterValues.__init__(self, values, chemistry)
     73 if values is not None:
     74     # If base_parameters is a filename, load from that filename
     75     if isinstance(values, str):
---> 76         file_path = self.find_parameter(values)
     77         path = os.path.split(file_path)[0]
     78         values = self.read_parameters_csv(file_path)

File ~\AppData\Local\Programs\Python\Python310\lib\site-packages\pybamm\parameters\parameter_values.py:755, in ParameterValues.find_parameter(path)
    753     if os.path.isfile(trial_path):
    754         return trial_path
--> 755 raise FileNotFoundError("Could not find parameter {}".format(path))

FileNotFoundError: Could not find parameter Chen2020

Failing that, I want to find the parameter values used for the model I have just run, I took it from tutorial 1:

import numpy as np
import pybamm
model = pybamm.lithium_ion.DFN()
sim = pybamm.Simulation(model)
sim.solve([0, 3600])

Ideally, I would like to simply like to change the Current and plot the terminal voltage along with the OCV curve to see how they compare.

**Comment by brosaplanella** [comment]
Which PyBaMM version are you using? Also, it seems you are using Python 3.10, which we do not support yet, unfortunately (only Python 3.8 and 3.9).

**Reply by MatHunt** [reply]
All I did was do pip install pybamm, do I need to be specific in what I am downloading?

**Reply by brosaplanella** [reply]
In theory this should install the most recent version, but there might be some issues if you are using Python 3.10. To check the version, run the following commands in Python:
```python3
import pybamm
print(pybamm.__version__)
```

**Reply by MatHunt** [reply]
I'm currently trying to remove python 3.10 and installing python 3.9 and starting the install process all over again.

**Comment by valentinsulzer** [comment, marked as answer]
See #2408 , reinstalling should fix it (but yeah python 3.10 also doesn't work)

**Reply by MatHunt** [reply]
"downgrading" to python 3.9 seemed to make everything work.
